src/routes/ethereum-navigator/+page.server.ts

```
@@ -0,0 +1,113 @@
+                headers: {
+                    'Authorization': `Bearer ${BEEHIIV_API_KEY}`,
+                    'Accept': 'application/json',
+                    'Content-Type': 'application/json'
+                },
+                signal: AbortSignal.timeout(5000)
+            }
+        );
+
+        if (!response.ok) {
+            const errorData = await response.json() as BeehiivErrorResponse;
+            const apiError = logError('Beehiiv API', new Error('API request failed'), {
```

ESLint

[error] 'apiError' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)